.github/workflows/deploy-release.yml

```diff
@@ -27,10 +27,25 @@ on:
         default: true
         required: false
         type: boolean
+  workflow_call:
+    inputs:
+      environment:
+        type: string
+        default: dev
+      ref:
+        default: "dev"
+        description: "Tag, branch, or commit"
+        required: true
+        type: string
+      lambdaDeploy:
+        description: "Include Lambda in deployment?"
+        default: true
+        required: false
+        type: boolean
```

Comment:
Why is this modified? I don't see that the pipeline uses this new workflow?

Reply:
It uses this workflow when it deploys to staging.  
This is because dev_deploy.yml deploys a SHA
and deployment.yml doesn't include the lambdas
so to deploy a tag to staging I needed to add workflow_call to this file.